Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -123,6 +152,15 @@ public List<DataPoint> Points2
         /// closed polygon.</remarks>
         public bool Reverse2 { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value indicating whether two color mode should be used.
+		/// </summary>
+		/// <value><c>true</c> if two color mode should be used; otherwise, <c>false</c>.</value>
+		/// <remarks> When using two color mode, Points and Points2 collections are considered
+		/// as different areas. They are painted with different colors and ConstantY2 is used to divide
+		/// the polygons horizontally. </remarks>
+		public bool TwoColorMode { get; set; }
```

Comment:
But as commented in #819, I think the two color mode belongs in `TwoColorAreaSeries`...